src/react-imgix.jsx

```diff
@@ -94,10 +96,26 @@ function buildSrc({
     srcSet = src;
   } else {
     if (fixedSize || type === "source") {
-      const dpr2 = constructUrl(rawSrc, { ...srcOptions, dpr: 2 });
-      const dpr3 = constructUrl(rawSrc, { ...srcOptions, dpr: 3 });
-      const dpr4 = constructUrl(rawSrc, { ...srcOptions, dpr: 4 });
-      const dpr5 = constructUrl(rawSrc, { ...srcOptions, dpr: 5 });
+      const dpr2 = constructUrl(rawSrc, {
+        ...(disableQualityByDPR || { q: CONSTANTS.q_dpr2 }),
+        ...srcOptions,
+        dpr: 2
+      });
+      const dpr3 = constructUrl(rawSrc, {
+        ...(disableQualityByDPR || { q: CONSTANTS.q_dpr3 }),
+        ...srcOptions,
+        dpr: 3
+      });
+      const dpr4 = constructUrl(rawSrc, {
+        ...(disableQualityByDPR || { q: CONSTANTS.q_dpr4 }),
+        ...srcOptions,
+        dpr: 4
+      });
+      const dpr5 = constructUrl(rawSrc, {
+        ...(disableQualityByDPR || { q: CONSTANTS.q_dpr5 }),
+        ...srcOptions,
+        dpr: 5
+      });
       srcSet = `${dpr2} 2x, ${dpr3} 3x, ${dpr4} 4x, ${dpr5} 5x`;
```

Comment:
Wait. We don't include a `1x` URL in the `srcset` at all here? That seems wrong. Can we remedy that?

Reply:
For some reason, I assumed that's what the fallback `src` was for but now realize that's not the case.

Reply:
It's possible that some browsers would fall back to the `src` if they didn't find a DPR item in the `srcset` that they like, but we definitely shouldn't rely on that behavior.